src/gui/wizard/flow2authwidget.ui

```diff
@@ -60,15 +51,31 @@
        </property>
       </widget>
      </item>
+     <item>
+      <spacer name="verticalSpacer">
+       <property name="orientation">
+        <enum>Qt::Vertical</enum>
+       </property>
+       <property name="sizeHint" stdset="0">
+        <size>
+         <width>20</width>
+         <height>32</height>
+        </size>
+       </property>
+      </spacer>
+     </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
+         <family>Segoe UI</family>
          <pointsize>12</pointsize>
-         <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
+       <property name="styleSheet">
+        <string notr="true">color:&quot;#191919&quot;</string>
+       </property>
        <property name="text">
```

Comment:
don't think it's a good idea to force a particular colour for the title either, this will make it hard to read for users who prefer using the dark mode 

```suggestion
       </property>
       <property name="text">
```